docs/node-mixin/dashboards/node.libsonnet

```diff
@@ -4,253 +4,393 @@ local row = grafana.row;
           refresh='time',
+          includeAll=true,
+          multi=true,
+          allValues='.+',
+        );
+
+        local syslog = 
+        logPanel.new(
+          'syslog Errors',
+          datasource='$loki_datasource',
         )
-      )
-      .addRow(
-        row.new()
-        .addPanel(idleCPU)
-        .addPanel(systemLoad)
-      )
-      .addRow(
-        row.new()
-        .addPanel(memoryGraph)
-        .addPanel(memoryGauge)
-      )
-      .addRow(
-        row.new()
-        .addPanel(diskIO)
-        .addPanel(diskSpaceUsage)
-      )
-      .addRow(
-        row.new()
-        .addPanel(networkReceived)
-        .addPanel(networkTransmitted)
-      ),
-  },
+        .addTarget(
+          loki.target('{filename=~"/var/log/syslog*|/var/log/messages*", %(nodeExporterSelector)s, instance=~"$instance"} |~".+(?i)error(?-i).+"' % $._config)
+        );
+
+        local authlog = 
+          logPanel.new(
+            'authlog',
+            datasource='$loki_datasource',
+          )
+          .addTarget(
+            loki.target('{filename=~"/var/log/auth.log|/var/log/secure", %(nodeExporterSelector)s, instance=~"$instance"}' % $._config)
+          );
+
+        local kernellog = 
+          logPanel.new(
+            'Kernel logs',
+            datasource='$loki_datasource',
+          )
+          .addTarget(
+            loki.target('{filename=~"/var/log/kern.log*", %(nodeExporterSelector)s, instance=~"$instance"}' % $._config)
+          );
+          
+        local journalsyslog = 
+          logPanel.new(
+            'Journal syslogs',
+            datasource='$loki_datasource',
+          )
+          .addTarget(
+            loki.target('{transport="syslog", %(nodeExporterSelector)s, instance=~"$instance"}' % $._config)
+          );
+          
+        local journalkernel = 
+          logPanel.new(
+            'Journal Kernel logs',
+            datasource='$loki_datasource',
+          )
+          .addTarget(
+            loki.target('{transport="kernel", %(nodeExporterSelector)s, instance=~"$instance"}' % $._config)
+          );
+          
+        local journalstdout = 
+          logPanel.new(
+            'Journal stdout Errors',
+            datasource='$loki_datasource',
+          )
+          .addTarget(
+            loki.target('{transport="stdout", %(nodeExporterSelector)s, instance=~"$instance", unit=~"$unit"} |~".+(?i)error(?-i).+"' % $._config)
+          );
+
+        local lokiDirectLogRow = 
+          row.new(
+            'Loki Direct Log Scrapes'
+          )
+          .addPanel(syslog)
+          .addPanel(authlog)
+          .addPanel(kernellog);
+
+        local lokiJournalLogRow = 
+          row.new(
+            'Loki Journal Log Scrapes'
+          )
+          .addPanel(journalsyslog)
+          .addPanel(journalkernel)
+          .addPanel(journalstdout);
+
+        $._NodeDashboard
+        .addTemplate(lokiDatasourceTemplate)
+        .addTemplate(jobTemplate)      
```

Comment:
Is this ever used? I don't see any place that the job template var is referenced in a query.

While the linter *does* have this as a rule, there's no sense in having the template var if it doesn't impact the output. Feel free to remove it, and add a `.lint` file to ignore the job rule. I may actually need to improve the linter in this area.

Reply:
Yeah this is something I was thinking about, maybe the instance variable by itself is enough for what we need, which is correlating logs and metrics.

Reply:
It is used though the `nodeExporterSelector` variable on the queries. I was thinking about removing it, since the instance variable is enough for our goals. But it is inhenrited from the upstream mixin, so I kept it and plan to configure it as `job="$job"` on the integration, this way if the user wants to have multiple jobs he can.

Reply:
Right now it is fixed at `job="integrations/node_exporter"` on our integration.